### PR TITLE
Website tag suggestion endpoint

### PR DESCRIPTION
**Pull request by leogrim**
@stkem @andrewconner 
Endpoint similar to the extension's, for keep cards.

### CODE REVIEW

**Comment by leogrim** [comment]
Actually @yipingliao

File: kifi-backend/modules/shoebox/conf/shoebox.routes [hunk 1]
```diff
@@ -328,7 +328,8 @@ GET     /site/libraries/:id/keeps   @com.keepit.controllers.website.LibraryContr
 POST    /site/libraries/:id/keeps   @com.keepit.controllers.website.LibraryController.addKeeps(id: PublicId[Library])
 GET     /site/libraries/:id/collabs @com.keepit.controllers.website.LibraryController.getCollaborators(id: PublicId[Library], count: Int, offset: Int ?= 0)
 POST    /site/libraries/:id/keeps/delete   @com.keepit.controllers.website.LibraryController.removeKeeps(id: PublicId[Library])
-DELETE  /site/libraries/:id/keeps/:keepId   @com.keepit.controllers.website.LibraryController.removeKeep(id: PublicId[Library], keepId: ExternalId[Keep])
+DELETE  /site/libraries/:id/keeps/:k   @com.keepit.controllers.website.LibraryController.removeKeep(id: PublicId[Library], k: ExternalId[Keep])
+GET     /site/libraries/:id/keeps/:k/tags/suggest  @com.keepit.controllers.ext.ExtLibraryController.suggestTags(id: PublicId[Library], k: ExternalId[Keep],  q: String, n: Option[Int] ?= None)
```
Comment (2is10) [inline kifi-backend/modules/shoebox/conf/shoebox.routes]: wrong controller

**Comment by 2is10** [comment]
otherwise LGTM

File: kifi-backend/modules/shoebox/conf/shoebox.routes (hunk 1 above)
Comment (leogrim) [inline kifi-backend/modules/shoebox/conf/shoebox.routes]: oups

**Pull request merged**